test/core/client_channel/lb_policy/lb_policy_test_lib.h

```diff
@@ -651,10 +738,6 @@ class LoadBalancingPolicyTest : public ::testing::Test {
     return &it->second;
   }
 
-  void ExpectQueueEmpty(SourceLocation location = SourceLocation()) {
```

Comment:
Please keep it. I use it a lot while developing test cases to debug some test issues.

Reply:
I didn't remove it; I just moved it up to line 453, next to the other similar methods.